.cirrus.yml

```diff
@@ -152,6 +152,13 @@ task:
         CHROME_NO_SANDBOX: true
         GOLD_SERVICE_ACCOUNT: ENCRYPTED[3afeea5ac7201151c3d0dc9648862f0462b5e4f55dc600ca8b692319622f7c3eda3d577b1b16cc2ef0311b7314c1c095]
       script:
+        - git clone https://github.com/flutter/web_installers.git
+        - cd web_installers/packages/web_drivers
+        - pub get
+        - dart lib/web_driver_installer.dart chromedriver --install-only
+        - chromedriver/chromedriver --port=4444 &
```

Comment:
why are we adding new tests to cirrus? I think we should add this one to LUCI, I added the details for this to go/flt-infra-web-long-tests

@godofredoc we are not adding new tasks to Cirrus CI now right, when is our ETA for flutter/flutter migration to LUCI?

Reply:
Another reason I don't want the test in this location is the issue @yjbanov filed: https://github.com/flutter/flutter/issues/67569

As stated in Cirrus CI, Chrome is not hermetic. This tests depends on both chrome and chromedriver. It's best to have it LUCI, since on LUCI the builder is hermetic. We already have these dependencies ready.

Reply:
That is correct, migration of the SDK is complete. We are moving one last test this week and all the new tests should go directly to LUCI and no new tests should be added to Cirrus.

Reply:
This is also already running on LUCI https://cs.opensource.google/flutter/recipes/+/master:recipe_modules/adhoc_validation/resources/web_smoke_test.sh are reason to added to Cirrus?

Reply:
I changed it. This is now a new shard called "web_long_running_tests" initially containing 3 subshards. PTAL.

Reply:
great! I added more comments to the how to doc, let me know if you need mode details.